src/UIKit/UIApplication.cs

```diff
@@ -128,6 +128,26 @@ public static UIContentSizeCategory ToEnum (NSString key)
 			// Do not compare to Medium, it is the fallback
 			return UIContentSizeCategory.Medium;
 		}
+
+		public static NSString FromEnum (UIContentSizeCategory category)
```

Comment:
Perhaps for an upcoming enumeration, we already have the enumeration and the values defined for this type from a previous version.